tests/integration/pilot/vm_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pilot
+
+import (
+	"strings"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
+	"istio.io/istio/pkg/test/framework/label"
+)
+
+func GetAdditionVMImages() []string {
+	// Note - bionic is not here as its the default
+	return []string{"app_sidecar_ubuntu_xenial", "app_sidecar_ubuntu_focal",
+		"app_sidecar_debian_9", "app_sidecar_debian_10", "app_sidecar_centos_8"}
+}
+
+func TestVmOSPost(t *testing.T) {
+	framework.
+		NewTest(t).
+		Features("traffic.reachability").
+		Label(label.Postsubmit).
```

Comment:
https://github.com/istio/istio/pull/25432

Individual tests in the pilot/ package should either utilize multiple clusters (see ) or have RequiresSingleCluster until they're converted. 

I need to document this somewhere/make it known. I think an integration testing best practices guide would be good; examples for sharing suite-wide echo instances, making your tests exercise all clusters etc and examples of things that can cause long test times.

Reply:
Ack, will add. FWIW the existing test I am moving did not have this either

Reply:
Yeah it's just the root of pilot/ that has multicluster allowed at the suite level.

Reply:
I see the issue, thanks. I fixed this now